client/web/compose/src/views/Admin/Pages/Edit.vue

```diff
@@ -376,7 +376,7 @@
                     :icon="['fas', 'plus']"
```

Comment:
There are two + signs here, since you changed the text to include the + and there was an icon here previously.
We should move away from including + in the text.
The wrapper should have the icon and the text Add in the button, just like Edit Page Layouts table has now on base branch